examples/arm/run.sh

```diff
@@ -25,12 +25,14 @@ target="ethos-u55-128"
 output_folder_set=false
 output_folder="."
 build_only=false
+portable_kernels="aten::_softmax.out"
 
 help() {
     echo "Usage: $(basename $0) [options]"
     echo "Options:"
     echo "  --model_name=<MODEL>                   Model to run, can be a builtin, examples/models or a filename Default to all builtin models"
     echo "  --aot_arm_compiler_flags=<FLAGS>       Only used if --model_name is used Default: ${aot_arm_compiler_flags}"
+    echo "  --portable_kernels=<OPS>               Comma separated list of portable (non delagated) kernels to include Default: ${portable_kernels}"
```

Comment:
this can blow up in size as the other issue was saying, we can take in a yaml which should also allow you to specify dtype etc.